clrdefinitions.cmake

```diff
@@ -1,5 +1,14 @@
 include(clrfeatures.cmake)
 
+# If set, indicates that this is not an officially supported release
+# Keep in sync with IsPrerelease in dir.props
+set(PRERELEASE 1)
```

Comment:
It seems like this is hard coded to 1. Shouldn't this be controlled by an environment variable or something?

Reply:
We want this to be always enabled in development branches and always disabled in release branches. Unfortunately, we couldn't find any other way. Environment variables wouldn't satisfy that requirement.